static/styles/application.css

```diff
@@ -0,0 +1,6025 @@
+.govuk-link {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+/*! Copyright (c) 2011 by Margaret Calvert & Henrik Kubel. All rights reserved. The font has been customised for exclusive use on gov.uk. This cut is not commercially available. */
```

Comment:
is this needed - aren't you pulling this in via node modules?

Reply:
This is the generated CSS from the sass file.  We need to add this to the git-ignore list.